docs2/site/docs/migrations/migration5.md

```diff
@@ -500,6 +500,50 @@ When adding a field by name only, such as `Field<StringGraphType>("Name");`, and
 rather than a property on the source object, the method parameters are parsed similarly to `FieldDelegate`
 as noted above with support for query arguments, `IResolveFieldContext`, `[FromServices]` and so on.
 
+### 16. Schemas can be entirely constructed from CLR types
+
+A new builder method `AddAutoSchema` has been added to allow building a schema entirely from CLR types
+using the new features within the auto-registering graph types to build the schema. Below is a sample:
+
+```csharp
+// sample configuration of DI
+var services = new ServiceCollection();
+services.AddGraphQL(b => b
+    .AddAutoSchema<Query>(s => s.WithMutation<Mutation>())
+    .AddSystemTextJson());
+var provider = services.BuildServiceProvider();
+
+
+// sample execution from DI
+var result = await provider.GetRequiredService<IDocumentExecuter>().ExecuteAsync(o =>
+{
+    o.RequestServices = provider;
+    o.Schema = provider.GetRequiredService<ISchema>();
+    o.Query = "{hero}";
+});
+var resultString = provider.GetRequiredService<IGraphQLTextSerializer>().Serialize(result);
+// resultString returns the following JSON: {"data":{"hero":"Luke Skywalker"}}
+
+
+// sample schema
+private class Query
```

Comment:
I suggest change private to public for this example.

Reply:
Good idea.

Reply:
done